.gitignore

```diff
@@ -84,3 +84,4 @@ tests/cases/user/puppeteer/puppeteer
 tests/cases/user/axios-src/axios-src
 tests/cases/user/prettier/prettier
 .eslintcache
+*v8.log
```

Comment:
a question, if I may, what kind of tools you were using to investigate those things? I imagine that in the case of Nodes the issue was already known for some time but I wonder how I could test those things in my libraries.

Reply:
I have an internal tool I wrote to analyze log files generated by various V8 commandline options.

Reply:
@Andarist One open source tool that might be of interest is https://github.com/thlorenz/deoptigate